### PR TITLE
Implement updated metadata view

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- adjust metadata view to desings
- adjust part of resources views

**Related issue(s)**
https://github.com/kyma-project/busola/issues/2753

### CODE REVIEW

File: src/resources/CronJobs/CronJobDescription.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Trans } from 'react-i18next';
+import { Link } from '../../shared/components/Link/Link';
+
+export const description = (
```
Comment (akucharska) [inline src/resources/CronJobs/CronJobDescription.js:4]: This file is not used. Leave only one file with description

File: src/resources/CronJobs/CronJob.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Trans } from 'react-i18next';
+import { Link } from '../../shared/components/Link/Link';
+
+export const description = (
```
Comment (akucharska) [inline src/resources/CronJobs/CronJob.js:4]: Leave only one file with description

File: src/resources/DaemonSets/DaemonSet.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Trans } from 'react-i18next';
+import { Link } from '../../shared/components/Link/Link';
+
+export const description = (
+  <Trans i18nKey="daemon-sets.description">
+    <Link
```
Comment (akucharska) [inline src/resources/DaemonSets/DaemonSet.js:6]: Leave only one file with description

File: src/resources/DaemonSets/DaemonSetDescription.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { Trans } from 'react-i18next';
+import { Link } from '../../shared/components/Link/Link';
+
+export const description = (
```
Comment (akucharska) [inline src/resources/DaemonSets/DaemonSetDescription.js:4]: Leave only one file with description

File: src/resources/Deployments/DeploymentConsts.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { Trans } from 'react-i18next';
+import { Link } from '../../shared/components/Link/Link';
+
+export const referenceDescription = (
```
Comment (akucharska) [inline src/resources/Deployments/DeploymentConsts.js:4]: Leave only one file with description

File: src/resources/Deployments/DeploymentDescription.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { Trans } from 'react-i18next';
+import { Link } from '../../shared/components/Link/Link';
+
+export const description = (
```
Comment (akucharska) [inline src/resources/Deployments/DeploymentDescription.js:4]: Leave only one file with description

File: src/shared/components/DynamicPageComponent/DynamicPageComponent.js [hunk 7]
```diff
@@ -166,28 +164,13 @@ export const DynamicPageComponent = ({
...
-                      open={showTitleDescription}
-                      onAfterClose={() => setShowTitleDescription(false)}
-                      placementType="Right"
-                    >
-                      <Text className="description">{description}</Text>
-                    </Popover>,
-                    document.body,
-                  )}
-                </>
+                <HintButton
```
Comment (akucharska) [inline src/shared/components/DynamicPageComponent/DynamicPageComponent.js:167]: Button does not align with the text. Try wrapping than in `FlexBox` like so:
```
<FlexBox>
                {title}
                {description && (
                  <HintButton
                    style={spacing.sapUiTinyMargin}
                    setShowTitleDescription={setShowTitleDescription}
                    showTitleDescription={showTitleDescription}
                    description={description}
                    context="dynamic"
                  />
                )}
              </FlexBox>
```

**Review by akucharska — CHANGES_REQUESTED** [review]
Very good job. I left some small suggestions in the comments.

File: src/resources/DaemonSets/DaemonSetDescription.js (hunk 4 above)
Comment (dbadura) [reply src/resources/DaemonSets/DaemonSetDescription.js:4]: Adjusted

File: src/resources/CronJobs/CronJobDescription.js (hunk 1 above)
Comment (dbadura) [reply src/resources/CronJobs/CronJobDescription.js:4]: Adjusted

**Review by akucharska — APPROVED** [review]

File: src/shared/components/ReadableElapsedTimeFromNow/ReadableElapsedTimeFromNow.tsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
+import { t } from 'i18next';
```
Comment (mrCherry97) [inline src/shared/components/ReadableElapsedTimeFromNow/ReadableElapsedTimeFromNow.tsx:1]: Never used

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: src/shared/components/ReadableElapsedTimeFromNow/ReadableElapsedTimeFromNow.tsx (hunk 8 above)
Comment (dbadura) [reply src/shared/components/ReadableElapsedTimeFromNow/ReadableElapsedTimeFromNow.tsx]: Removed

**Review by akucharska — APPROVED** [review]

**Comment by dbadura** [comment]
retest

**Pull request merged**